components/channel/internal/channel/channel.go

```diff
@@ -148,23 +156,73 @@ func (c *Channel) processSuccessesAndErrors() {
 	}()
 }
 
+// Map CloudEvent Context Headers To Kafka Message Headers
+func (c *Channel) createProducerHeaders(context cloudevents.EventContext) []kafka.Header {
+	headers := []kafka.Header{
+		{Key: "ce_specversion", Value: []byte(context.GetSpecVersion())},
+		{Key: "ce_type", Value: []byte(context.GetType())},
+		{Key: "ce_source", Value: []byte(context.GetSource())},
+		{Key: "ce_id", Value: []byte(context.GetID())},
+		{Key: "ce_time", Value: []byte(context.GetTime().Format(time.RFC3339))},
+	}
+
```

Comment:
Should we have constants for these header names?  Or does the Cloud Event library have such constants?